static/js/app/campaign_results.js

```diff
@@ -2,6 +2,20 @@ var map = null
 
 // statuses is a helper map to point result statuses to ui classes
 var statuses = {
+    "HTML Opened": {
+        slice: "ct-slice-donut-opened",
+        legend: "ct-legend-opened",
+        label: "label-danger",
+        icon: "fa-file",
+        point: "ct-point-opened"
+    },
+    "Doc Opened": {
```

Comment:
There are two kinds of attachments in the program, which are html and ms word. One of them is called "HTML Opened". Do you think it is OK if I changed "Doc Opened" to "WORD Opened"?